.eslintrc.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
                     },
                     {
                         selector: ['parameter', 'method'],
-                        format: ['camelCase'],
+                        format: ['camelCase', 'PascalCase'],
```

Comment:
NAB - why this change?

Reply:
Sure thing, let me explain! So when it comes to lines 9-11 (and typing HOCs in general) we are following guidelines from these two sources:

- [forwardRef TypeScript guidelines](https://www.carlrippon.com/react-forwardref-typescript/)
- [HOC TypeScript guidelines](https://react-typescript-cheatsheet.netlify.app/docs/hoc/full_example/)

it's the same approach in all TypeScript HOC PRs, that's something we agreed upon. About `PascalCase`, it's necessary because in HOCs components (`WrappedComponent`) are passed as function parameters.

Hope that answers your questions, if there is anything else you'd like me to explain let me know!

Reply:
ooh got it. Cool, that makes sense!